### ISSUE TITLE
Make an additional "misc" section in the comparison with information about the libraries other than features

### ISSUE THREAD

**Issue by meew0** [DiscordAPI/libraries#10]
This would be useful because features aren't the only thing to compare. These are some sections I propose:
- Documentation
  - In-line (like ESDoc, YARD, JavaDoc etc.)
  - Explicit (method documentation explicitly written out, like discord.js has)
  - Tutorials
  - maybe a value that rates the coverage of the documentation
- Examples
- Implementation details
  - Are tokens cached?
  - Is it async/threaded? If yes, how much and what parts?
  - ...
- Extensions (I guess these are technically features, but unrelated to API support)
  - Command parsing
  - Temporary event handlers added on-the-fly, I call these "awaits", discord.js has "awaitResponse"
  - (others)
- Security
  - Does REST use HTTPS?
  - Is wss used?
  - Are cached tokens encrypted? If yes, how strong?
  - Voice encryption?
- A field that displays the current amount of GitHub stars, using one of these fancy always-up-to-date image providers

Please propose other sections here you might find useful or necessary!

**Comment by Auralytical** [comment]
FYI - HTTPS and WSS are mandatory (or will be very soon)

**Comment by hlaaftana** [comment]
They're mandatory already.

**Comment by hlaaftana** [comment]
rip

**Comment by meew0** [comment]
I saw this in my notifications and thought "finally somebody commented on this" but alas...

**Comment by bwmarrin** [comment]
Well, I think it's a good idea.  Just need to make a nice list.  I bet if you've got an idea you could do a PR against the repo to add another section :)  Ideally you would want to make sure whatever is listed is more fact-based and avoid subjective things.

**Comment by meew0** [comment]
To be fair, the current way is also a little subjective (Does "Yes" in a REST feature mean the user can do it at all using the library, or does it have to be nicely abstracted? Does sending messages include TTS? Does voice sending include transcoding? ...) I see what you mean though. Is there any particular thing in my list that's subjective in your opinion?

**Comment by bwmarrin** [comment]
No, just a general thought.  I think there's a few on my list that are subjective :). Like, "Does this lib suck" or is "lib code is hella messy"

**Comment by hlaaftana** [comment]
feature comparison charts are always subjective though

**Comment by abalabahaha** [comment]
Yes implies an abstraction. Nobody cares about TTS :stuck_out_tongue: 

The list may get messy since lib devs will start clamoring to get/remove features to make their libs look better on the charts.

**Comment by meew0** [comment]
In that case, maybe we should get rid of the extensions part and focus on the other things. Maybe turn "Implementation details" into an "API compliancy" section that keeps track of token caching, on-demand chunking, rate limit respection and so on.

**Comment by abalabahaha** [comment]
If the comparison is user oriented, there's not much point in putting API compliancy

If the devs are using this instead of keeping milestones/checklists/issues/reminders/Trellos/etc... pls

**Comment by hlaaftana** [comment]
its been 2 and a half months should we close this yet

**Comment by abalabahaha** [comment]
no

@uniquoooo stop abusing comment-editing

**Comment by davidcole1340** [comment]
oops

**Comment by meew0** [comment]
After thinking about it for two and a half months, I still think it's a good idea

**Comment by suicvne** [comment]
i second that

**Comment by hlaaftana** [comment]
ok then fork and add everything and open a pr even though it seems like some of your suggestions are discordrb favored

**Comment by AraHaan** [comment]
I think it is a good idea too @meew0.

**Comment by hlaaftana** [comment]
How tf did you find this

**Comment by abalabahaha** [comment]
GitHub recently introduced a feature called "reactions". Instead of making a comment to signal your reaction to a comment, you could simply click the reactions button and react to the comment.

**Comment by bwmarrin** [comment]
+1

**Comment by AraHaan** [comment]
@hlaaftana ez I am god. :P

**Comment by hlaaftana** [comment]
Yeah thats just common knowledge

**Comment by hlaaftana** [comment]
So who's gonna do this

**Comment by abalabahaha** [comment]
haven't even agreed on what fields should be added lol

**Comment by meew0** [comment]
I'll do it

**Comment by meew0** [comment]
I'll make a PR and then we'll see

**Comment by hlaaftana** [comment]
Close?

**Comment by meew0** [comment]
Actually, abal still needs to update the layout, so I'll reopen it and keep it open until that's done

**Comment by abalabahaha** [comment]
no

EDIT: latency and ^

**Comment by meew0** [comment]
wow that was fast
